models/issue_dependency.go

```diff
@@ -8,12 +8,12 @@ import (
 	"time"
 )
 
-// IssueDependency is connection request for receiving issue notification.
+// IssueDependency represents an issue dependency
 type IssueDependency struct {
 	ID           int64     `xorm:"pk autoincr"`
-	UserID       int64     `xorm:"UNIQUE(watch) NOT NULL"`
-	IssueID      int64     `xorm:"UNIQUE(watch) NOT NULL"`
-	DependencyID int64     `xorm:"UNIQUE(watch) NOT NULL"`
+	UserID       int64     `xorm:"NOT NULL"`
+	IssueID      int64     `xorm:"NOT NULL"`
+	DependencyID int64     `xorm:"NOT NULL"`
```

Comment:
`IssueID` and `DependencyID` should still have `UNIQUE(blah)` over them

Reply:
Done.